sdk/servicebus/azservicebus/errors.go

```diff
@@ -1,7 +1,13 @@
 package azservicebus
 
-import "errors"
+import "fmt"
 
-var ErrSenderClosed = errors.New("sender is closed and cannot be used")
-var ErrReceiverClosed = errors.New("receiver has been closed and can no longer be used")
-var ErrProcessorClosed = errors.New("processor has been closed and can no longer be used")
+// implements `internal/errorinfo/NonRetriable`
+type ErrClosed struct {
```

Comment:
I don't think this needs to be exported.

Reply:
You are right. Unexported :)